File: TemplateRcView/app/src/main/java/com/example/templatercview/lib_recycleview/Renderer.java
package com.example.templatercview.lib_recycleview;

import android.view.View;

import androidx.recyclerview.widget.RecyclerView;

public interface Renderer {

    void onBinding(RecyclerView.ViewHolder holder, int position);
    Holder onGenerate(View view);
    int onCounting();
}

File: StockChart2/app/src/main/java/com/example/stockchart2/Renderer.kt
package com.example.stockchart2

import android.view.View
import androidx.recyclerview.widget.RecyclerView.ViewHolder

interface Renderer {
    fun onAssignNotifying(notify: Notifier?)
    fun onGenerate(view: View?): TypeViewHolder?
    fun onBinding(holder: ViewHolder?, position: Int)
    fun onCounting(): Int
}